frontend/src/components/pages/UserProfile/NickName.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react";
+
+import EditTwoToneIcon from '@material-ui/icons/EditTwoTone';
+
+import ChangeNickName from "./NickNameChange";
+
+export default function NickName({handleClick}) {
+  const [nickName, setNickName] = useState([
+    {
+      name: "nick_name",
+    },
+  ]);
+
+  console.log("else",handleClick)
```

Comment:
This console add no value, good to use for testing locally, but should not be commited